homeassistant/components/homewizard/sensor.py

```diff
@@ -108,98 +108,98 @@ class HomeWizardSensorEntityDescription(
     ),
     HomeWizardSensorEntityDescription(
         key="total_power_import_kwh",
```

Comment:
Keys are unaltered to prevent the need of migration. Maybe do this in a follow-up (or not at all?)

Reply:
Yeah doing that in a followup is fine 👍